tests/event_store/test_named_streams.py

```diff
@@ -57,10 +57,10 @@ class CorruptedStreamId(StreamId):
 
 
 def test_kurrentdb_cant_use_category_with_dash(
-    kurrentdb: BackendFactory,
+    kurrentdb: Backend,
```

Comment:
How about calling this fixture `kurrentdb_backend`?

Reply:
Ok, I will rename it

Reply:
Do you mean adding `_backend` to all backend fixtures?